viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -250,11 +256,44 @@ export class LighthouseReportViewer {
       features.initFeatures(json);
     } catch (e) {
       logger.error(`Error rendering report: ${e.message}`);
-      container.textContent = '';
+      root.textContent = '';
       throw e;
     } finally {
       this._reportIsFromGist = this._reportIsFromPSI = this._reportIsFromJSON = false;
     }
+  }
+
+  /**
+   * @param {LH.FlowResult} json
+   * @param {HTMLElement} root
+   */
+  _renderFlowResult(json, root) {
+    renderFlowReport(json, root);
+    // Install as global for easier debugging.
+    window.__LIGHTHOUSE_FLOW_JSON__ = json;
```

Comment:
This same global is used in the standalone report. Should we just reuse `__LIGHTHOUSE_JSON__` in the viewer? Otherwise the console could be printing a mix of `__LIGHTHOUSE_JSON__` and `__LIGHTHOUSE_FLOW_JSON__`.

Reply:
Keeping them separated is my preference